azure-templates/stages.yml

```diff
@@ -22,6 +22,7 @@ parameters:
   # buildSteps: list of {projectLocation, buildOperation, target, buildSpec}
   - name: buildSteps
     type: object
+    default: 'None'
```

Comment:
Did you consider an empty string? No strong preference from me, but I'm not sure what prior art does for things like this.

Reply:
the rest of stages.yml uses '' so I agree using that is more consistent.  We use actual words in some locations, but only for readability of the build outputs in azure (this will not result in any build outputs)
3bd71e0396cbc624e276ee53d64f65f937278dd2